packages/expo-modules-core/src/uuid/uuid.web.ts

```diff
@@ -3,16 +3,17 @@ import v35 from './lib/v35';
 import { UUID, Uuidv5Namespace } from './uuid.types';
 
 function uuidv4(): string {
-  // Crypto needs to be required when run in Node.js environment.
-  const cryptoObject =
-    typeof crypto === 'undefined' || typeof crypto.randomUUID === 'undefined'
-      ? require('crypto')
-      : crypto;
-
-  if (!cryptoObject?.randomUUID) {
-    throw Error("The browser doesn't support `crypto.randomUUID` function");
+  if (
+    // Node.js has supported global crypto since v15.
+    typeof crypto === 'undefined' &&
+    // Only use abstract imports in server environments.
+    typeof window === 'undefined'
+  ) {
+    // eslint-disable-next-line no-eval
+    return eval('require')('node:crypto').randomUUID();
```

Comment:
Maybe add a comment why you're using `eval` here

Reply:
good idea